plugins/modules/net_tools/nmcli.py

```diff
@@ -572,6 +585,8 @@ def __init__(self, module):
         self.type = module.params['type']
         self.ip4 = module.params['ip4']
         self.gw4 = module.params['gw4']
+        self.routes4 = module.params['routes4']
+        self.never_default4 = module.params['never_default4']
```

Comment:
Don't you have to convert it to `yes` / `no` if it is not `None` now?

Reply:
For what I understand, this part is done here:

```
    @staticmethod
    def settings_type(setting):
        if setting in ('bridge.stp',
                       'bridge-port.hairpin-mode',
                       'connection.autoconnect',
                       'ipv4.never-default'):
            return bool
        elif setting in ('ipv4.dns',
                         'ipv4.dns-search',
                         'ipv4.routes',
                         'ipv6.dns',
                         'ipv6.dns-search'):
            return list
        return str
```